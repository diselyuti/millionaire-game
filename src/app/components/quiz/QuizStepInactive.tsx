import React from 'react';
import styles from '@/app/styles/components/quiz/quiz-step.module.scss';

function QuizStepInactive({ text }: { text: string }) {
  return (
    <article className={styles.step}>
      <p>{text}</p>
    </article>
  );
}

export default QuizStepInactive;
