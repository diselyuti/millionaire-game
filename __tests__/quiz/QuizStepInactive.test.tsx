import React from 'react';
import '@testing-library/jest-dom';
import QuizStepInactive from "@/app/components/quiz/QuizStepInactive.tsx";
import {render} from "@testing-library/react";

describe('QuizStepInactive', () => {
  test('renders component correctly', () => {
    const { getByText } = render(<QuizStepInactive text="$100" />);
    expect(getByText('$100')).toBeInTheDocument();
  });

  test('should match snapshot', () => {
    const { container } = render(<QuizStepInactive text="$100" />);
    expect(container).toMatchSnapshot();
  });
});
