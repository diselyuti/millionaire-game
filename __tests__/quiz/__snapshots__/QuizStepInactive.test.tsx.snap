// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`QuizStepInactive should match snapshot 1`] = `
<div>
  <article
    class="step"
  >
    <p>
      $100
    </p>
  </article>
</div>
`;
